wagtail/admin/userbar.py

```diff
@@ -63,6 +63,36 @@ class AccessibilityItem(BaseItem):
     #: For more details, see `Axe documentation <https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter-examples>`__.
     axe_rules = {}
 
+    #: A list used to add custom rules to the existing set of Axe rules, or to override the properties of existing Axe rules
+    #: For more details, see `Axe documentation <https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axeconfigure>`__.
+    axe_custom_rules = []
+
+    #: A custom rule to check the quality of the Images alt texts. Added to the list of custom rules and enabled by default.
+    #: Should be used in conjunction with :attr:`_axe_custom_alt_text_check`
+    #: This rule ensures that alt texts doesn't contain antipatterns like file extensions. Returns zero false positives.
+    _axe_custom_alt_text_rule = [
+        {
+            "id": "alt-text-quality",
+            "impact": "serious",
+            "selector": "img[alt]",
+            "tags": ["best-practice"],
+            "any": ["check-image-alt-text"],
+            "enabled": True,  # If ommited, defaults to True
+        }
+    ]
+
+    #: A list used to add custom checks to the existing set of Axe checks, or to override the properties of existing Axe checks
+    #: For more details, see `Axe documentation <https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axeconfigure>`__.
+    axe_custom_checks = []
+
+    #: A custom check for the custom Image alt text quality rule. Added to the list of custom checks and enabled by default.
+    #: Should be used in conjunction with :attr:`_axe_custom_alt_text_rule`
+    _axe_custom_alt_text_check = [
+        {
+            "id": "check-image-alt-text",
```

Comment:
From the [axe.configure API](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#api-name-axeconfigure), this is where we need to define an `options` dict with our regex pattern defined as a string. Something like:

```suggestion
            "id": "check-image-alt-text",
            "options": { "pattern": "\.(png|jpeg|gif)" },
```